blivet/formats/luks.py

```diff
@@ -130,7 +130,7 @@ def __init__(self, **kwargs):
         self._key_file = kwargs.get("key_file")
         self.escrow_cert = kwargs.get("escrow_cert")
         self.add_backup_passphrase = kwargs.get("add_backup_passphrase", False)
-        self.min_luks_entropy = kwargs.get("min_luks_entropy", 0)
+        self.min_luks_entropy = kwargs.get("min_luks_entropy", luks_data.min_entropy)
```

Comment:
If this is a real minimum maybe we should be treating as such by doing something like `min(kwargs.get('min_luks_entropy') or 0, luks_data.min_entropy)`.

Reply:
Oops, had that backwards.

Reply:
It's not really an absolutely required minimum, it's more a recommended minimum.